src/makefile

```diff
@@ -1,10 +1,13 @@
 PREFIX?=/usr/local
 LIBS=-lsodium
 DEFINES=
-CFLAGS=-march=native -Wall -O2 -g -fstack-protector-strong -D_FORTIFY_SOURCE=2 -fasynchronous-unwind-tables -fpic -fstack-clash-protection -fcf-protection=full -Werror=format-security -Werror=implicit-function-declaration -Wl,-z,defs -Wl,-z,relro -ftrapv -Wl,-z,noexecstack $(DEFINES)
+CFLAGS?=-march=native -Wall -O2 -g -fstack-protector-strong -D_FORTIFY_SOURCE=2 -fasynchronous-unwind-tables -fpic -fstack-clash-protection -fcf-protection=full -Werror=format-security -Werror=implicit-function-declaration -Wl,-z,defs -Wl,-z,relro -ftrapv -Wl,-z,noexecstack $(DEFINES)
 LDFLAGS=-g $(LIBS)
 CC=gcc
 SOEXT=so
+AEXT=a
```

Comment:
what would be the aext on windows?

Reply:
In the Visual Studio toolchain, it would be .lib.
However, in the gcc toolchain (e.g. in MinGW), it is also .a.

Reply:
ah, right. should've remembered .lib